### ISSUE TITLE
Error on websocket: Invalid WebSocket frame: invalid status code 1005

### ISSUE THREAD

**Issue by EmMoxxChen** [schulz3000/deepstreamNet#56]
Hello, when I started the ConsoleSample, I could not get the result. The following exception information was found on the server side. How can I deal with this problem?
INFO | Heartbeat missing on websocket, terminating connection
ERROR | Error on websocket: Invalid WebSocket frame: invalid status code 1005

**Comment by schulz3000** [comment]
Hi @EmMoxxChen,
which version of the deepstream server do you run?
Is there any further output on the client side?

Do you have another client, e.g. the javascript client connected in parallel? If yes, did you receive the event sent from the ConsoleSample?

**Comment by EmMoxxChen** [comment]
I am very glad to receive your reply. 
The Deepstream Server version is 5.2.4. 
I did not run the JavaScript client, but I tried it, and JavaScript can respond successfully.
The client also cannot receive events that it sends out.

ConsoleSample client didn't have any response when executing the statement "var challengeResult = await registerAuthChallenge (credentials).configureAawait (false);", the program will not go down, and there will be no error.
